porcelain/miner.go

```diff
@@ -283,6 +285,7 @@ func MinerPreviewSetPrice(ctx context.Context, plumbing mpspAPI, from address.Ad
 // mgoaAPI is the subset of the plumbing.API that MinerGetOwnerAddress uses.
 type mgoaAPI interface {
 	MessageQuery(ctx context.Context, optFrom, to address.Address, method string, params ...interface{}) ([][]byte, error)
+	ActorGetSignature(ctx context.Context, actorAddr address.Address, method string) (*exec.FunctionSignature, error)
```

Comment:
Not blocking, but it would better match our established pattern to create new interfaces for `MinerGetSectorSize` and `MinerGetLastCommittedSectorID`.